packages/services/api/src/modules/integrations/resolvers/Organization.ts

```diff
@@ -50,14 +49,7 @@ export const Organization: Pick<
       },
     });
   },
-  viewerCanModifySlackIntegration: async (organization, _arg, { session, injector }) => {
-    const isAvailable = await injector.get(SlackIntegrationManager).isAvailable({
-      organizationId: organization.id,
-    });
-    if (!isAvailable) {
-      return false;
-    }
```

Comment:
Slack integration can not be disabled. `isAvailable` checks if a GitHub integration exists for the organization...